UNIXFS.md

```diff
@@ -59,6 +64,27 @@ For files that are comprised of more than a single block, the 'Type' field will
 
 This data is serialized and placed inside the 'Data' field of the outer merkledag protobuf, which also contains the actual links to the child nodes of this object.
 
+## Metadata
+
+There are three ways to specify file metadata:
+
+1. The repeated `metadata` field in a directory applies metadata to each file in the directory.
+2. An intermediary node with a `Type` of `Metadata` applies metadata to an individual file. While this feature should be supported, it has been deprecated.
```

Comment:
I left this in here because we technically support it. However, I'd be happy to remove it if it's not in-use. go-ipfs has some internal support but it's hard to _actually_ use this feature so I doubt any one is doing so (and it doesn't work properly in many cases).

Reply:
...and completely unspecified, does `Data` contain the metadata or would it now be the metadata field?

(Not expecting that question to be answered - just stating the state of affairs 😉)

Reply:
`Data` contains the metadata object. Note: this entire thing was completely unspecified until this spec was written...